src/database/config/config.js

```diff
@@ -0,0 +1,29 @@
+const dotenv = require('dotenv')
+dotenv.config()
+
+const commonDatabaseConfig = {
+  dialect: 'postgres',
+  dialectOptions: {
+    ssl: {
+      require: true,
+      rejectUnauthorized: false,
+    },
+  },
+}
```

Comment:
why do we have two database configurations

Reply:
Yes , we have  2  database configurations due to this following factors:
1)we are allowed only to run migrations if the config file has an extension of .js ,If not we get this error.
![image](https://github.com/atlp-rwanda/e-commerce-bitcrafters-bn/assets/106761137/61cf08db-6991-4154-931f-eadea8d62b6d)

2)And again if the database config file has an extension of .js, it is not be supported to be imported/invoked and be used everywhere it is imported like in **index.ts** , and in  models , Because the  creation of new  instance of **sequelize** is done in js can not be called and be used in typescript files.  that is reason why we have sequelizer configurations in typescript.
And also that **db.config.ts** helps us to switch  the database is for development or testing to facilitate database switching .

We tried to create new sequelizer object in js but when it is invoked in **index.ts** to establish the connection but not works and throws an error.